WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -854,8 +855,8 @@ private void populateSelectedCategories() {
         LayoutInflater layoutInflater = getActivity().getLayoutInflater();
 
         if (mCategories != null) {
+            AppCompatButton buttonCategory = (AppCompatButton) layoutInflater.inflate(R.layout.category_button, null);
```

Comment:
This needs to be moved back inside the `for` loop or the app crashes when there are more than two categories.